Mathlib/LinearAlgebra/CliffordAlgebra/Basic.lean

```diff
@@ -351,6 +351,20 @@ theorem ι_range_map_map (f : Q₁ →qᵢ Q₂) :
   (ι_range_map_lift _ _).trans (LinearMap.range_comp _ _)
 #align clifford_algebra.ι_range_map_map CliffordAlgebra.ι_range_map_map
 
+/-- If a linear map preserves the quadratic forms and is surjective, then the algebra
+maps it induces betweem Clifford algebras is also surjective.-/
+lemma map_surjective {Q₁ : QuadraticForm R M₁} {Q₂ : QuadraticForm R M₂} (f : Q₁ →qᵢ Q₂)
+    (hf : Function.Surjective f) : Function.Surjective (CliffordAlgebra.map f) := by
+  rw [← LinearMap.range_eq_top, LinearMap.range_eq_map, Submodule.eq_top_iff']
```

Comment:
I think this is just making work for yourself; can't you work with the definition of `Surjective` directly?